participants/Dragonfly video system.md

```diff
@@ -0,0 +1,104 @@
+
+The membership users watch videos on PC browsers or dedicated APP.creators are able to upload videos or set sharing price by client tool. if owners don't like to share their videos,they can also take them off the shelf.the APP will show them the numbers of sharing time and the digital encrypted currency they have.
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+*You can reference the [list of curated datasets in this repo](https://github.com/filecoin-project/slingshot/blob/master/datasets.md). If you'd like to use a dataset that is not on that list, please nominate your chosen dataset by updating the curated datasets table in this PR. If nominating a dataset, do you own or have exclusive rights to the data you plan to use?*
+
+Not curated dataset.
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+*What sorts of data will you collect? Do you own or have exclusive rights to the data you plan to use? Is the data license available online? Does the data contain any sensitive or illegal information?*
+
+We will collect video files. and these video files are uploaded to IPFS by owners.We are authorized to have exclusive rights to the data we plan to use.the data license is availabe online. No license required, No sensitive and illegal information. 
+
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+
+*Give us as realistic an estimate as you possibly can at this stage. We understand it's impossible to have perfect information before really starting your project!*
+
+about 100TB.
+
+## (7) How are you structuring the data?
+
+*Is this structured or unstructured data? What data formats, schemas, etc. does this dataset follow?*
+
+Those video data is Unstructured data, the data formats are MP4, AVI,WMV,etc.usually We will convert most video files in MP4 file format.
+
+## (8) What pre-processing are you doing before ingesting the data?
+
+*How will you prepare this data for ingestion into Filecoin? What size storage deals will you be making? If this is a tabular dataset or directory structure, how will you maintain indices into the data so you can retrieve specific data as needed for your application? This is currently one of the most important steps to being successful with data storage on Filecoin. Feel free to ask for help and/or look at docs for recommendations!*
+
+just download the video file from cameras or other storage systems.by file format  conversion,some files need to be compressed in advanced video compression algorithm.usually,each video file size is less than 1 GB.so each file could be sealed into the IPFS sectors smoothly.
+
+Not tabular dataset or directory structure.
+
+## (9) What tech stack will you use to store the data?
+
+*Either:* lotus
```

Comment:
How will you be handling retrieval?   You may want to consider Powergate or IPFS for hot retrieval.